LICENSE

```diff
@@ -1,21 +1,10 @@
-MIT License Copyright (c) 2020 Tierney Cyren
+MIT License
 
-Permission is hereby granted, free
-of charge, to any person obtaining a copy of this software and associated
-documentation files (the "Software"), to deal in the Software without
-restriction, including without limitation the rights to use, copy, modify, merge,
-publish, distribute, sublicense, and/or sell copies of the Software, and to
-permit persons to whom the Software is furnished to do so, subject to the
-following conditions:
+Copyright (c) 2020 Contributors https://github.com/pkgjs/design/graphs/contributors
```

Comment:
I hope this is fine with you @bnb?

Just going through all pkgjs repos and standardizing various docs.